### PR TITLE
Pca 23/create topic resource edit view

### PR DESCRIPTION
**Pull request by MaslaLT**

### CODE REVIEW

File: app/controllers/TopicController.php [hunk 1]
```diff
@@ -17,14 +18,22 @@ public function show($id)
         }
     }
 
-    public function edit($id)
-    {
-        echo "Topic edit - $id";
+    public function edit($id) {
+        {
+            $topic = Topic::find($id);
```
Comment (Icybrew) [inline app/controllers/TopicController.php:24]: No error checking, what if topic is not found?

File: app/controllers/TopicController.php [hunk 2]
```diff
@@ -17,14 +18,22 @@ public function show($id)
...
+            $docTags = DB::table("doctags")->getAll();
+
+            $this->view('topic/editTopic', [
```
Comment (Icybrew) [inline app/controllers/TopicController.php]: Rename editTopic to edit

**Review by Icybrew — CHANGES_REQUESTED** [review]

File: resources/views/topic/edit.php.twig [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            <div class="form-group">
+                <label for="HtmlRemarks">HtmlRemarks:</label>
+                <textarea rows="15" class="form-control" id="HtmlRemarks" name="RemarksHtml">{{ topic.RemarksHtml }}</textarea>
+            </div>
+            <button type="submit" class="float-right btn btn-success">Submit Changes</button>
+        </form>
+        <button class="btn btn-dark" onclick="history.go(-1);"> ← Back</button>git
+
+    </div>
+{% endblock %}
```
Comment (indruteok) [inline resources/views/topic/edit.php.twig]: add new line

**Review by indruteok — APPROVED** [review]

File: resources/views/topic/edit.php.twig [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+                    {% endfor %}
+                </select>
+            </div>
+            <div class="form-group">
+                <label for="HtmlRemarks">HtmlRemarks:</label>
+                <textarea rows="15" class="form-control" id="HtmlRemarks" name="RemarksHtml">{{ topic.RemarksHtml }}</textarea>
+            </div>
+            <button type="submit" class="float-right btn btn-success">Submit Changes</button>
+        </form>
+        <button class="btn btn-dark" onclick="history.go(-1);"> ← Back</button>git
```
Comment (Icybrew) [inline resources/views/topic/edit.php.twig]: git?

File: resources/views/topic/edit.php.twig [hunk 5]
```diff
@@ -0,0 +1,34 @@
+{% extends "template.php.twig" %}
+
+{% block content %}
+    <div class="container">
+
+        <!-- Topic Edit Form -->
+        <form class="mt-5" method="post" action="../{{ topic.Id }}">
```
Comment (Icybrew) [inline resources/views/topic/edit.php.twig]: use route() function in action

**Review by Icybrew — CHANGES_REQUESTED** [review]

**Review by Icybrew — APPROVED** [review]

**Pull request merged**